sterror/sterror.go

```diff
@@ -0,0 +1,157 @@
+package sterror
+
+import (
+	"errors"
+)
+
+type Op string
+
+func (o Op) String() string {
+	return string(o)
+}
+
+type Scope string
+
+func (s Scope) String() string {
+	return string(s)
+}
+
+type Info string
+
+func (i Info) String() string {
+	return string(i)
+}
+
+const (
+	colon     = ": "
+	hyphen    = " - "
+	separator = "\n"
+)
+
+const (
+	Undefined Scope = "Undefined"
+	Host      Scope = "Host"
+	Network   Scope = "Network"
+	Opts      Scope = "Opts"
+	Ospkg     Scope = "Ospkg"
+	Stlog     Scope = "Stlog"
+	Trust     Scope = "Trust"
+)
+
+/* Error-Wrapper
+- op: Function that raised the error
+- Scope: Package that raised the error
+- Info: additional Info passed as a string
+- Err: Wrapped underlying error.
+*/
```

Comment:
Its more common to place the explanations of the fields at their lines and a general description at the top like: 
```
// Foo is for doing cool stuff
type Foo struct {
    Bar Bar // Bar ist for ...
    ...
}
```

Reply:
And although it is a multiline comment, for some reason usually there is just `//` on every line.